src/app/_core/models/dtos.ts

```diff
@@ -0,0 +1,36 @@
+export interface WeatherDto {
+  dt:string;
+  main:{
+    temp:number,
+    feels_like:number,
+    temp_min:number,
+    temp_max:number,
+    pressure: number,
+    sea_level: number,
+    grnd_level: number,
+    humidity: number,
+    temp_kf: number
+  };
+  weather:{id:number;
```

Comment:
Can you add a return? The display should look like the `main` object above.
PS: it would be nice to use a linter ;)